oat-sa/tao-test-runner-qti-fe
# Feature/TR-5209/Cleanup calculator engine

### Related to: https://oat-sa.atlassian.net/browse/TR-5209

Requires: 
 - [x] https://github.com/oat-sa/tao-core-libs-fe/pull/85
 - [x] https://github.com/oat-sa/tao-core-sdk-fe/pull/168
 - [x] https://github.com/oat-sa/tao-core-ui-fe/pull/554
 - [x] update the file [`package.json`](https://github.com/oat-sa/tao-test-runner-qti-fe/blob/b8bcbf4a9dcf440a46319099abd857fbc593070d/package.json#L58-L60) after the libraries have been released.

### Summary

Update the calculator for using the extracted engine [tao-calculator](https://github.com/oat-sa/tao-calculator-fe).

### Details

Update the libraries

### How to test

- install: `npm i`
- run the tests: `npm test`
- check the companion PR
